project-base/src/Shopsys/ShopBundle/Resources/scripts/styleguide/styleguide.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Script used for the Styleguide structure only
+ */
+
+if (typeof window !== 'undefined') global = window;
```

Comment:
Where is this script from - is it MIT or some other opensource license?

Reply:
it is minified group of files from this project: https://hugeinc.github.io/styleguide/

Reply:
Styleguide is MIT as well
https://github.com/hugeinc/styleguide/blob/master/LICENSE.md